src/views/ReturnDetails.vue

```diff
@@ -268,6 +269,10 @@ export default defineComponent({
       this.queryString = ''
     },
     async scanCode () {
+      if (!hasCameraAccess()) {
+        showToast(translate("Camera access is required to scan items."));
```

Comment:
As we are having a toast message in the `hasCameraAccess` function, so we do not need to have another toast here. From here we can simply return the flow without any toast or log.